google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1414,14 +1416,16 @@ public Builder setDefaultEventBasedHold(Boolean defaultEventBasedHold) {
       return this;
     }
 
+    /** @deprecated Use {@link #setRetentionEffectiveTimeOffsetDateTime(OffsetDateTime)} */
     @Override
     @Deprecated
     Builder setRetentionEffectiveTime(Long retentionEffectiveTime) {
-      return setRetentionEffectiveTime(millisOffsetDateTimeCodec.encode(retentionEffectiveTime));
+      return setRetentionEffectiveTimeOffsetDateTime(
```

Comment:
nit: inconsistencies between style.
before you do 
```
Builder setUpdateTime(Long updateTime) {
      this.updateTime = millisOffsetDateTimeCodec.encode(updateTime);
      return this;
    }
```
vs calling the offset method in the deprecated old method.